Loop/Views/SettingsView.swift

```diff
@@ -196,8 +189,6 @@ extension SettingsView {
     private var serviceChoices: [ActionSheet.Button] {
         var result = viewModel.servicesViewModel.inactiveServices.map { availableService in
             ActionSheet.Button.default(Text(availableService.localizedTitle)) {
-                // TODO: this "dismiss then call didTapAddService()" here is temporary, until we've completely gotten rid of SettingsTableViewController
-                self.dismiss()
                 self.viewModel.servicesViewModel.didTapAddService(availableService)
```

Comment:
When I tap to add the service and successfully add one, the view doesn't refresh (I have to exit the settings screen and then re-open it for the changes to show up)